src/utils/wallet.ts

```diff
@@ -7,6 +7,7 @@
 
 
 import { crypto, HDPublicKey, HDPrivateKey, Address } from 'bitcore-lib';
+import { createHash, HexBase64Latin1Encoding } from 'crypto';
```

Comment:
This is a leftover from a refactor I did, went unnoticed because we don't have a linter on this project, thanks!!

FIxed on 337d8273966a2086143b6eea5b143dfad050d43c

Reply:
Great! Thanks!